### DISCUSSION TITLE
Question: Can istio be configured to use a second http/https proxy?

### DISCUSSION THREAD

**Discussion by xiaoxuanyo** [istio/istio#51029] [Q&A]
Description:
I am using istio as an sidecar proxy. There is a use case where the whole network is behind an https proxy. Is there a way to have istio redirect to this second proxy?

i.e. client->istio sidecar (http/https, xxx.com)->[http proxy2]-> xxx.com

Does istio honor the HTTP_PROXY/HTTPS_PROXY/NO_PROXY environment variables? Is there another way?

**Comment by escoffier** [comment]
Looks like you need is a egress gateway

**Reply by xiaoxuanyo** [reply]
What it can help me solve is to unify export traffic, but can it help me forward http/https to my http proxy service (supports HTTP over connect). My understanding is as follows:

k8s pod  -> mesh -> egress gateway -> [http proxy2] -> xxx.com ？


From the information I know, VirtualService can only forward the traffic of certain hosts, but what I want to achieve is that all http/https in this pod must go through my clash client proxy service before accessing the target website. , so I want to use EnvoyFilter to achieve it, but I really understand too little about this. If you can help me, I will be very grateful~~~
